core/src/consensus/stake/mod.rs

```diff
@@ -428,6 +454,20 @@ pub fn ban(state: &mut TopLevelState, criminal: Address) -> StateResult<Deposit>
         (_, Some(jailed)) => jailed.deposit,
         _ => 0,
     };
+    // confiscate criminal's deposit and give the same deposit amount to the informant.
+    if let Some(criminal_info) = candidates.get_candidate(&criminal).map(Clone::clone) {
+        candidates.remove(&criminal);
+        if let Some(informant_info) = candidates.get_candidate(&public_to_address(informant)).map(Clone::clone) {
+            candidates.add_deposit(
```

Comment:
Why give CCC as a deposit?
Any reason for this? How about give CCC directly to the informant?

Reply:
Yes you are right, I misread the spec